js/src/personal/tony/Tony.page.tsx

```diff
@@ -1,35 +1,41 @@
-import { Title, Text } from '@mantine/core'
+// TonyPage component
+
+import { Title, Text, Grid, Image } from '@mantine/core'
 import styles from './Tony.module.css'
 
 export function TonyPage() {
   return (
     <>
-      <div className={styles.background}>
-        <Title
-          className={styles.title}
-          ta="left"
-          mt={50}
-        >
-          <Text
-            inherit
-            variant="gradient"
-            component="span"
-            gradient={{ from: 'blue', to: 'green' }}
+      <Grid
+        gutter="md"
+        className={`${styles.grid}`}
+      >
+        <Grid.Col span={6}>
+          <Title
+            className={styles.title}
+            ta="left"
           >
-            Tony Wang
+            <Text
+              inherit
+              component="span"
+            >
+              {'Tony Wang'}
+            </Text>
+          </Title>
+          <Text className={styles.text}>{'Software Engineer Intern'}</Text>
+          <Image
+            className={styles.pfp}
+            src="https://raw.githubusercontent.com/mantinedev/mantine/master/.demo/images/bg-7.png"
```

Comment:
We should self-host our own images for your websites. What happens if mantine decides to change this url?